src/webgpu/api/operation/rendering/basic.spec.ts

```diff
@@ -109,5 +109,24 @@ g.test('fullscreen_quad').fn(async t => {
 });
 
 g.test('large_draw')
-  .desc(`Test reasonably-sized large {draw, drawIndexed} (see also stress tests).`)
+  .desc(
+    `Test reasonably-sized large {draw, drawIndexed} (see also stress tests).
+
+  Tests that draw calls behave reasonably with large vertex counts for
+  non-indexed draws, large index counts for indexed draws, and large instance
+  counts in both cases. Various combinations of these counts are tested with
+  both direct and indrect draw calls.
+
+  Draw call sizes are increased incrementally over these parameters until the
+  completion of a draw call exceeds a fixed time limit of 500ms.
```

Comment:
Hopefully this actually works well! We'll have to monitor the test once it rolls out to automated testing to make sure it isn't too flaky or anything.